testsuites/syncgateway/functional/topology_specific_tests/multiple_sync_gateways/test_cache_management_multple_sgws.py

```diff
@@ -121,7 +122,7 @@ def test_sgw_cache_management_multiple_sgws(params_from_base_test_setup):
             pytest.skip('XATTR tests require --xattrs flag')
         sg_conf_name = "sync_gateway_default_functional_tests"
     else:
-        sg_conf_name = "xattrs/no_import"
+        sg_conf_name = "c"
```

Comment:
I am not able to find an sg config file named like c_cc.json under resources/sync_gateway_configs. is this a typo or the sg config missed from the change list?

Reply:
oops,  looks like while fixing flake8 issue, it got replaced with c by mistakenly . Replaced back with right one